scripts/functions/installer

```diff
@@ -1019,150 +1021,175 @@ setup_user_profile_check()
 setup_user_profile_detect()
 {
   etc_profile_file="/etc/profile.d/rvm.sh"
-  search_list_mksh=( "$HOME/.mkshrc" "$HOME/.profile" )
-  search_list_bash=( "$HOME/.bashrc" "$HOME/.bash_profile" "$HOME/.bash_login" )
-  search_list_zsh=( "${ZDOTDIR:-${HOME}}/.zshenv" "${ZDOTDIR:-${HOME}}/.zprofile" "${ZDOTDIR:-${HOME}}/.zshrc" "${ZDOTDIR:-${HOME}}/.zlogin" )
```

Comment:
Any particular reason for limiting the list, my idea with all the files was that user can have sourcing/PATH line in any of them as long as it solves their needs, then the code was detecting which entries were already set up and not installing needlessly in other file which we think is more appropriate for it.

Reply:
In newer UNIXes aren't used anymore